pipes/WDL/tasks/tasks_assembly.wdl

```diff
@@ -263,18 +361,23 @@ task refine_assembly_with_aligned_reads {
           --outVcf ${sample_name}.sites.vcf.gz \
           --min_coverage ${min_coverage} \
           --major_cutoff ${major_cutoff} \
-          ${"--NOVOALIGN_LICENSE_PATH=" + novocraft_license} \
           --JVMmemory "$mem_in_mb"m \
           --loglevel=DEBUG
 
         file_utils.py rename_fasta_sequences \
           refined.fasta "${sample_name}.fasta" "${sample_name}"
+
+      # collect figures of merit
+      grep -v '^>' refined.fasta | tr -d '\n' | wc -c | tee assembly_length
+      grep -v '^>' refined.fasta | tr -d '\nNn' | wc -c | tee assembly_length_unambiguous
     }
 
     output {
-        File   refined_assembly_fasta  = "${sample_name}.fasta"
-        File   sites_vcf_gz            = "${sample_name}.sites.vcf.gz"
-        String viralngs_version        = read_string("VERSION")
+        File   refined_assembly_fasta       = "${sample_name}.fasta"
```

Comment:
Should we be consistent in how we're naming the "final" output?
Here we use `refined_assembly_fasta` whereas in the denovo pipeline [we call it](https://github.com/broadinstitute/viral-pipelines/blob/0258ed5f482a0291d5f23e620cb633f8ed6f7337/pipes/WDL/tasks/tasks_assembly.wdl#L461) `final_assembly_fasta `.

Reply:
We can, which way do you think we should go? The thing is, refine_2x_and_plot is actually doing the work of three Tasks all smushed into one, so it actually knows that the second refine is the *final* one, whereas if you just called refine twice, the task can't really know.

I haven't done it in this PR, but if we like this general direction and want to move the denovo workflows this way too, it would probably also imply breaking up the refine_2x_and_plot to look more like this one: scatter the alignments, merge, refine, scatter alignments, merge, refine, plot.

Reply:
The latter is probably the way to go. I'm mostly concerned about fasta file names ending up in the right column when dumped from DNAnexus or entered into a Terra table. I guess we can match on `^.*_assembly_fasta$`.

Reply:
Good point. That said, starting from scratch in a new Terra workspace for production analyses, we’ll probably stick to the refbased and not be using the denovo as much